scss/_accordion.scss

```diff
@@ -58,39 +56,36 @@
 }
 
 .accordion-item {
+
   &:first-of-type {
+
     .accordion-button {
       @include border-top-radius($accordion-border-radius);
```

Comment:
wrong variable value according to bootstrap need to be `accordion-inner-border-radius`and so lint won't cry for unusued variable